.github/workflows/ci.yml

```diff
@@ -17,40 +17,20 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        with_rust: ['WITH_RUST=ON', 'WITH_RUST=OFF']
+        with_rust: ['true', 'false']
```

Comment:
This PR is awesome. Thanks!

My only concern is this change. This string is used to display the test matrix so now it lookslike `(windows, true)` which is not obvious what it's testing.

Let's customize the name https://futurestud.io/tutorials/github-actions-customize-the-job-name so the test matrix is more readable.